packages/terra-menu/src/Menu.scss

```diff
@@ -0,0 +1,104 @@
+:local {
+  /* stylelint-disable selector-class-pattern */
+  .submenu + .arrow[data-align-bottom='true']::after {
+    border-top-color: var(--terra-menu-submenu-background-color, #f9f9f9);
+  }
+
+  .divider {
+    border-color: var(--terra-menu-divider-color, #dedfe0);
+    border-style: solid;
+    border-width: 1px 0 0;
+    margin-top: 2px;
+
+    &::after {
+      border-color: var(--terra-menu-divider-shadow-color, #f9f9f9);
+      border-style: solid;
+      border-width: 3px 0 0;
+      content: '';
+      display: block;
+      margin-bottom: 4px;
+    }
+  }
+
+  .submenu .divider::after {
+    border-color: var(--terra-menu-submenu-divider-shadow-color, #fff);
+  }
+
+  .header {
+    background-color: var(--terra-menu-submenu-header-background-color, #fff);
+    border-bottom: var(--terra-menu-submenu-header-border-bottom, 1px solid #dedfe0);
+    width: 100%;
+
+    .header-title {
+      color: var(--terra-menu-submenu-header-title-color, #1c1f21);
+      font-size: var(--terra-menu-submenu-header-title-font-size, 12px);
+      font-weight: normal;
+      line-height: var(--terra-menu-submenu-header-title-line-height, 1.1667);
+      margin: var(--terra-menu-submenu-header-title-margin, 7px 0 7px 0);
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
I'll have to leave that up to @neilpfeiffer. During initial design discussions we decided that the title could be truncated since the user would see the full string on the previous menu page, where the item was clicked. I don't have a problem with making it opt-in if UX agrees.